docs/integrations.md

```diff
@@ -0,0 +1,16 @@
+Atomist receives events from and performs operations on many systems,
+including version control, CI systems, Slack, and more.
+
+Atomist uses the native integration technology for each platform or
+tool.  For example, to integrate with GitHub and Travis CI, Atomist
+uses webhooks; to integrate with Slack it uses their native real-time
+messaging (RTM) API.  For each platform Atomist integrates with, it
```

Comment:
We no longer use RTM, we use their event API, which I think is also webhook driven.

Reply:
great!